setup.py

```diff
@@ -44,6 +44,9 @@
         'stratify',
         'vmprof',
         'yamale',
+        'sklearn',
```

Comment:
Please also add the dependencies to meta.yml so they get included in the conda package.